adaptors/library/jobs/satusehat-create-condition.js

```diff
@@ -0,0 +1,97 @@
+// Create a diagnosis in satusehat
+
+fn(state => {
+  state.conditionMappedData = state.visit.diagnosis.flatMap(diagnosis =>
+    state.lookup_table.dentist_diagnosis
```

Comment:
What is `state.lookup_table` and where did it come from? It feels like an important thing to explain. Or at least mention?